sysadmin/phpmyadmin/config.inc.php

```diff
@@ -0,0 +1,157 @@
+<?php
```

Comment:
I think this file is included as `sysadmin/phpmyadmin/config.inc.php` and `phpmyadmin/config.inc.php`.

Reply:
Deleted!